pseudoanonymize/anonymization.py

```diff
@@ -0,0 +1,92 @@
+from typing import Any, Dict
+
+from .base import BaseProcessor
+from .common import make_chat_completion
+from .exceptions import UnparsableLLMOutputException
```

Comment:
Maybe just personal preference but I'd opt for absolute imports - historically they've been more robust to different interpreter setups, etc.